zenodo/config.py

```diff
@@ -73,6 +73,12 @@ def _(x):
     """Identity function for string extraction."""
     return x
 
+
+#: System sender email address
+ZENODO_SYSTEM_SENDER_EMAIL = 'system@zenodo.org'
+#: Email address of admins
+ZENODO_ADMIN_EMAIL = 'zenodo-admin@cern.ch'
```

Comment:
maybe keep the default names (besides license etc.), strictly zenodo-specific, but not cern-specific? e.g.: 'zenodo-admin@zenodo.org` and move the cern email to the instance config?